Basic/Pod/PP.pod

```diff
@@ -506,6 +365,170 @@ be a PDL developer (and even then it's not obligatory)
 
 =back
 
+=head1 FUNCTIONS
+
+Here is a quick reference list of the functions provided by PDL::PP.
+
+=head2 pp_add_boot
+
+=for ref
+
+Add code to the BOOT section of generated XS file
+
+
+=head2 pp_add_exported
+
+=for ref
+
+Add functions to the list of exported functions
+
+
+=head2 pp_add_isa
+
+=for ref
+
+Add entries to the @ISA list
+
+
+=head2 pp_addbegin
+
+=for ref
+
+Sets code to be added at the top of the generate .pm file
+
+
+=head2 pp_addhdr
+
+=for ref
+
+Add code and includes to C section of the generated XS file.
+
+When used in a module that is "multi-C" (one F<.c> file per C<pp_def>ed
+function), you need to bear in mind that as each one is generated, all the
+C<pp_addhdr> so far will be included. Therefore, if you add C functions,
+make sure to make them C<static> to avoid clashes with later F<.c> files.
+But a better practice is make them be separate C files, with any necessary
+F<.h> to be included by them and the F<.pd> file. You can then add them
+to your F<Makefile.PL> (note this is the C<_int> version, see separate
+notes on how to "opt-in" for your own modules):
```

Comment:
This should probably mention the new `CHeader` option, that adds C text only to the individual `.c` file. Could you add it?